README.md

```diff
@@ -103,15 +105,38 @@ With the following settings you can:
    3. Update the external report called DEFAULT_REPORT_SERVICE. It should have a URL of: https://portal-report.concord.org/branch/master/index.html?sourceKey=app.lara.docker.username **Replace `username` with the username on your local system. If you don't know your username, run `echo $USER`**.
 4. in LARA: `cp .env-osx-sample .env`
 5. in LARA `.env`:
-    1. set `REPORT_SERVICE_TOKEN` (see the comment in the .env file)
+    1. set `REPORT_SERVICE_URL` (see the value in the .env-osx-sample file)
+    1. set `REPORT_SERVICE_TOKEN` (see the comment in the .env-osx-sample file)
     2. set `LARA_PROTOCOL=https`
     3. set `PORTAL_PROTOCOL=https`
```

Comment:
You probably want to number these 1-4 (or make them all 1s so they auto-increment).

Reply:
Thanks @kswenson. I realized I actually added this to the wrong section.